graphql/bin/create_and_push_docker_image.sh

```diff
@@ -0,0 +1,43 @@
+#!/bin/bash
+## =============================================================================
+## NAME: build.sh
+##
+##
+## DESCRIPTION
+## -----------------------------------------------------------------------------
+## Builds the GraphQL image and pushes it to NASA Github Packages.
+##
+## USAGE
+## -----------------------------------------------------------------------------
+## bin/create_and_push_docker_image.sh
+##
+## This must be called from the (root) graphql directory /graphql
+##
+## Args:
+##   $1 - The version number to create in Github.
+##   $2 - An access token with the `write:packages` permission on the NASA organization.
+## =============================================================================
+
+## Set this for Debugging only
+#set -ex
+
+## Make sure we are calling the script the correct way.
+BASEDIR=$(dirname $0)
+if [ "$BASEDIR" != "bin" ]; then
+  >&2 echo "ERROR: This script must be called from the root directory of the component [bin/create_and_push_docker_image.sh]."
+  exit 1
+fi
+
+
+source ../bin/common/check_returncode.sh
+
+version_number=$1
+github_token=$2
+
+echo $github_token | docker login ghcr.io -u nasa --password-stdin
```

Comment:
Does -u HAS to be `nasa`?

Reply:
Yes, because it's pointing to the package repository. It is not strictly related to the user the token is assigned to.